apps/example/jest.config.esm.js

```diff
@@ -0,0 +1,10 @@
+const coreConfig = require("./jest.config");
+
+module.exports = {
```

Comment:
is this still needed if we only have jestSetup.js now?

Reply:
No it is just testing that previous setups won't fail

Reply:
got it. Can you add a comment for it. so we don't forget.